core-bundle/src/Resources/contao/widgets/SerpPreview.php

```diff
@@ -66,21 +66,23 @@ public function generate()
 		$aliasField = $this->getAliasField($suffix);
 		$descriptionField = $this->getDescriptionField($suffix);
 		$descriptionFallbackField = $this->getDescriptionFallbackField($suffix);
-		$titleTag = $model->getRelated('layout')->titleTag;
```

Comment:
This leads to a possible uninitialized variable on line 102, doesn't it?

Reply:
True. I was not checking this anymore, because I figured that we need a callback anyway. 🙈

Reply:
This would probably be easier if `new Contao.SerpPreview()` would just get a json_encoded associative array. Then you can omit values and this line would just get sth like `$data['titleTag'] = $model->getRelated('layout')->titleTag;`.